src/frontend/components/SøknadsSteg/VelgBarn/LeggTilBarn/LeggTilBarnModal.tsx

```diff
@@ -22,20 +23,21 @@ const LeggTilBarnModal: React.FC<{
     erÅpen: boolean;
     lukkModal: () => void;
 }> = ({ erÅpen, lukkModal }) => {
+    const { tekster } = useApp();
     const { skjema, nullstillSkjema, valideringErOk, leggTilBarn, validerFelterOgVisFeilmelding } =
         useLeggTilBarn();
-    const { tekster } = useApp();
 
-    const teksterForLeggTilBarnModal = tekster().FELLES.modaler.leggTilBarn;
+    const teksterForModal: ILeggTilBarnTekstinnhold = tekster().FELLES.modaler.leggTilBarn;
     const {
+        tittel,
         erBarnetFoedt,
         ikkeFoedtAlert,
         barnetsNavnSubtittel,
         fornavn,
         etternavn,
         foedselsnummerEllerDNummer,
         foedselsnummerAlert,
-    } = teksterForLeggTilBarnModal;
+    } = teksterForModal;
```

Comment:
Ser ut som de deconstructor variabelen her tidligere så kan vel bare gjøre det videre 
 
```suggestion
        ... , 
        leggTilKnapp
    } = tekster().FELLES.modaler.leggTilBarn;
```